R/get_columns.R

```diff
@@ -97,15 +97,15 @@
 #'   \item \code{col_type}: The type of column.
 #'   \item \code{refs}: The unique identification number(s) for the reference(s)
 #'     associated with the column.
-#'   \item \code{max_thick}: Maximum unit thickness in meters.
+#'   \item \code{max_thick}: The maximum unit thickness in meters.
 #'   \item \code{max_min_thick}: The maximum possible minimum thickness in
 #'     meters.
 #'   \item \code{min_min_thick}: The minimum possible minimum thickness in
 #'     meters.
-#'   \item \code{b_age}: The estimated bottom age of the column, in millions of
-#'     years before present.
-#'   \item \code{t_age}: The estimated top age of the column, in millions of
-#'     years before present.
+#'   \item \code{b_age}: The age of the bottom of the unit, estimated using the
+#'      continuous time age model, in millions of years before present.
```

Comment:
This should be changed to 'age of the bottom of the column', as get_columns() returns info for an entire column.